glue/core/subset.py

```diff
@@ -797,6 +797,9 @@ def to_mask(self, data, view=None):
     def copy(self):
         return RangeSubsetState(self.lo, self.hi, self.att)
 
+    def traverse(self):
```

Comment:
Hmm, I am a little surprised this is needed. The `__str__` on L1098 also traverse in a similar way but I don't see any custom `__str__` defined here.